consult-gh.el

```diff
@@ -1136,8 +1136,7 @@ If repo or targetdir are not supplied, interactively asks user for those values.
 (defun consult-gh-repo-fork (&optional repos)
 "Interactively forks the repository defined by `repo` to the current user account logged in with `gh` command line tool after confirming name. It uses `gh fork repo ...`."
   (interactive)
-  (let* ((consult-gh-prioritize-local-folder nil)
-         (repos (or repos (consult-gh--read-repo-name))))
+  (let* ((repos (or repos (consult-gh--read-repo-name))))
```

Comment:
instead of doing this, I edited consult-gh--read-repo-name, so the local repo can be selected by default by using `M-n (added to the end of history). I think that's a better implementation.

The issue with your solution is that if a user sets `consult-gh-prioritize-local-folder to t, then they have to be careful not to call `consult-gh-repo-fork from a folder they don't want to fork! Using that option for listing issues is fine because worse case scenario the user sees the issues of the wrong repo but forking and cloning, this is not the right way. 

In #57, I provide a better solution by having the local repo as the default input in completing-read.